code/mail/SESMailer.php

```diff
@@ -116,7 +116,7 @@ protected function sendMessage($destinations, $from, $subject, Mime\Message $bod
 		$rawMessageText = $this->getMessageText($message);
 		
 		if(strlen($rawMessageText) > 256 && class_exists('QueuedJobService')) {
-			singleton('QueuedJobService')->queueJob(Injector::inst()->get('SESQueuedMail', false, array(
+			singleton('QueuedJobService')->queueJob(Injector::inst()->createWithArgs('SESQueuedMail', array(
```

Comment:
Just an fyi, you can use `Injector::inst()->create('Name', $param1, $param2)` instead of createwith args.